### PR TITLE
Improve Delete Class with Sections

### PR DESCRIPTION
**Pull request by mudssrali**
This PR show improvements and bugfix for #553

### CODE REVIEW

File: web/src/actions/index.ts [hunk 1]
```diff
@@ -463,12 +463,23 @@ export const deleteClass =
...
+				...deletes,
+				{ path: ['db', 'settings', 'classes', 'additionalFees', misClass.id] },
+				{ path: ['db', 'settings', 'classes', 'defaultFee', misClass.id] },
+				{ path: ['db', 'classes', misClass.id] }
+			]
+		} else {
+			deletes = [
+				...deletes,
+				...Object.keys(misClass.sections).map(secId => ({
+					path: ['db', 'classes', misClass.id, 'sections', secId]
```
Comment (ZarishIqbal) [inline web/src/actions/index.ts:479]: You should save class information in class history before deleting it.

File: web/src/pages/classes/create.tsx [hunk 2]
```diff
@@ -530,17 +533,9 @@ const DeleteClassModal = ({
...
-	const totalStudents = useMemo(
-		() => getStudentsInClass(students, sectionIds).length,
-		[students, sectionIds]
-	)
-
 	const alertText = tipUser
-		? ''
-		: 'All fee settings for this class will also be gone. Are you sure you want to proceed?'
+		? 'Are you sure you want to proceed?'
+		: 'All fee settings for this class will also be removed. Are you sure you want to proceed?'
```
Comment (ZarishIqbal) [inline web/src/pages/classes/create.tsx:538]: Fee settings aren't removed in the case when all sections are not deleted.

**Review by ZarishIqbal — COMMENTED** [review]
I have made two suggestions. Kindly, check them out.

File: web/src/actions/index.ts (hunk 1 above)
Comment (mudssrali) [reply web/src/actions/index.ts:479]: We would implement this at some point.

File: web/src/pages/classes/create.tsx (hunk 2 above)
Comment (mudssrali) [reply web/src/pages/classes/create.tsx:538]: I intentionally left this. We will take this later.

**Pull request merged**